SeleniumTrainingCenter/BaseTest.cs

```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SeleniumTrainingCenter
+{
+    public class BaseTest
+    {
+        private static readonly IWebDriver _driver = new ChromeDriver();
```

Comment:
I recommend you to wrap web driver initialization into [SetUp] method. It will allow to initialize web driver instance before each test separately.